pkg/api/v1beta3/dynakube/feature_flags.go

```diff
@@ -129,9 +129,9 @@ func (dk *DynaKube) FeatureNoProxy() string {
 	return dk.getFeatureFlagRaw(AnnotationFeatureNoProxy)
 }
 
-// FeatureActiveGateAutomaticTLSCertificate is a feature flag to enable automatic creation of AG TLS certificate if TrustedCAs are used.
-func (dk *DynaKube) FeatureActiveGateAutomaticTLSCertificate() bool {
-	return dk.getFeatureFlagRaw(AnnotationFeatureActiveGateAutomaticTLSCertificate) == truePhrase
+// FeatureActiveGateDisableAutomaticTLSCertificate is a feature flag to disable automatic creation of AG TLS certificate.
+func (dk *DynaKube) FeatureActiveGateDisableAutomaticTLSCertificate() bool {
+	return dk.getFeatureFlagRaw(AnnotationFeatureActiveGateDisableAutomaticTLSCertificate) == truePhrase
 }
```

Comment:
```suggestion
func (dk *DynaKube) FeatureActiveGateAutomaticTLSCertificate() bool {
	return dk.getFeatureFlagRaw(AnnotationFeatureActiveGateAutomaticTLSCertificate) != falsePhrase
}
```

probably, this is the PR (+ test update)
- this just makes the FF `true` by default, and therefore an opt out

similar example:
```go
func (dk *DynaKube) FeatureAutomaticKubernetesApiMonitoring() bool {
	return dk.getFeatureFlagRaw(AnnotationFeatureAutomaticK8sApiMonitoring) != falsePhrase
}
```